docker-compose.yml

```diff
@@ -0,0 +1,290 @@
+---
+version: '2.1'
+
+services:
+    authentication-web:
+      image: docker.artifactory.reform.hmcts.net/auth/authentication-web:latest
+      environment:
+        - IDAM_API_URL=http://idam-api:8080
+        - IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_BULK_SCAN_ORCHESTRATOR=123456
+        - IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CCD_GATEWAY=123456
+        - IDAM_CONTINUE_URL_VALIDATOR_ENFORCE_TLS=false
+      mem_limit: 256m
+      memswap_limit: 0
+      depends_on:
+        idam-api:
+          condition: service_started
+      healthcheck:
+        disable: true
+      ports:
+        - 8000:8000
+
+    idam-api:
+      image: docker.artifactory.reform.hmcts.net/auth/idam-api
+      command: --wait-for-database 60
+      environment:
+        - JAVA_OPTS=-Xms8m -Xmx256m -Djava.security.egd=file:/dev/./urandom -XX:MaxMetaspaceSize=128m
+        - SPRING_DATASOURCE_URL=jdbc:postgresql://shared-database-v10:5432/idam
+        - IDAM_TESTING_SUPPORT_ENABLED=true
+        - SPRING_MAIL_HOST=smtp-server
+        - NOTIFY_API_KEY=AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        - NOTIFY=false
+        - NOTIFY_CMC_WELCOME_USER_TEMPLATE=fake
+        - NOTIFY_DIVORCE_WELCOME_USER_TEMPLATE=fake
+        - NOTIFY_SSCS_WELCOME_USER_TEMPLATE=fake
+        - NOTIFY_RESET_PASSWORD_TEMPLATE=fake
+        - NOTIFY_PROBATE_ACTIVATE_USER_TEMPLATE=fake
+        - NOTIFY_DIVORCE_ACTIVATE_USER_TEMPLATE=fake
+        - NOTIFY_SSCS_ACTIVATE_USER_TEMPLATE=fake
+        - NOTIFY_CCD_ACTIVATE_USER_TEMPLATE=fake
+        - NOTIFY_BAR_ACTIVATE_USER_TEMPLATE=fake
+        - NOTIFY_FREG_ACTIVATE_USER_TEMPLATE=fake
+        - IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CMC_CITIZEN=123456
+        - IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CMC_LEGAL=123456
+        - IDAM_API_OAUTH2_CLIENT_CLIENT_SECRETS_CCD_GATEWAY=123456
+        - http_proxy
+        - https_proxy
+        - no_proxy
+        - REFORM_ENVIRONMENT=test
+        - IDAM_BSO_SYSTEMUPDATE_USER=bulkscanorchestrator+systemupdate@gmail.com
+        - IDAM_BSO_SYSTEMUPDATE_PASSWORD=Password12
+      mem_limit: 512m
+      memswap_limit: 0
+      depends_on:
+        shared-database-v10:
+          condition: service_healthy
+      ports:
+        - 8080:8080
+
+    smtp-server:
```

Comment:
is this needed?

Reply:
`idam-api` needs smtp server `-SPRING_MAIL_HOST=smtp-server` else idam-api errors out